### ISSUE TITLE
Callback doesn't execute

### ISSUE THREAD

**Issue by Vishal0203** [twolfson/iframe2image#3]
The console.log inside iframe2image callback doesn't get called. There are no errors on browser

```
$('#get_image').on('click', function() {
	var canvas = document.getElementById('canvas'),
	context = canvas.getContext('2d');

	canvas.width = 300;
	canvas.height = 200;

	var iframe = document.getElementById('preview');
	console.log(iframe);

	iframe2image(iframe, function (err, img) {
		console.log(img);
		if (err) { return console.error(err); }
		context.drawImage(img, 0, 0);
	})
})
```

**Comment by twolfson** [comment]
We had a similar issue in #2 which was patched in `0.2.0`. Can you verify you're on the latest version?

**Comment by Vishal0203** [comment]
@twolfson The code on master is latest I suppose. I had to move all the content outside onLoad method to make it work.  https://github.com/twolfson/iframe2image/blob/master/dist/iframe2image.withdomvas.js#L106

**Comment by twolfson** [comment]
Ah, we updated `lib` but forgot to update `dist`. We'll fix that up by the end of the week

**Comment by twolfson** [comment]
Okay, we've recompiled `dist` and released it under `0.2.1`. Sorry for the inconvenience!
